react.md

```diff
@@ -86,17 +137,73 @@ FTC was based on [React Boilerplate](https://github.com/react-boilerplate/react-
   // Webpack config, generators etc.
   internals
   ```
+  
+## React: Boilerplate
+
+Need to get up and running, and want more than just `create-react-app?` FTC was based on [React Boilerplate](https://github.com/react-boilerplate/react-boilerplate), which has changed a bit since we started using it. [Introduction into the boilerplate](https://github.com/react-boilerplate/react-boilerplate/blob/master/docs/general/introduction.md) is a great place to start.
+
+
+
+## React: Styling
+
+### Preferred: CSS Modules
+ 
+#### CSS Modules Resources
+
+We’re using SCSS syntax with CSS being loaded by webpack as [CSS modules](https://github.com/css-modules/css-modules).
+
+SCSS builtins are available, as are most language features like variables. Mixins are provided by CSS Modules using composiion and work differently from SCSS mixins. Refer to the [css-modules documentation](https://github.com/css-modules/css-modules#dependencies) for examples.
+
+### Secondary: SASS 
+
+### Other Choices?
+
+- *Radium* is a popular choice with React, but team members have found it difficult to work with for larger apps. Radium works by injecting styling directly inline, as `style={}`, which can be difficult to override, even using the `!important` modifier. Often we've run into conflicting styles competing on one component.
+
+- *Semantic UI React* is a popular, quick styling solution. It's a library that comes with most styled components out of the box. However, any customization or deviation from their styles are very hard to implement well, to override their styling.
+
+## React: Testing
+
+### Preferred: Jest and Cypress
+
+[Jest](https://facebook.github.io/jest/) for unit testing
+https://github.com/react-boilerplate/react-boilerplate/blob/master/docs/testing/unit-testing.md
+
+[Cypress](https://www.cypress.io/)
+ 
+### Secondary: ??
+
+### Other Choices?
+
+
+## React: Linting Tools
+
+### Preferred ??
```

Comment:
eslint is definitely the winner here & many of our projects amke use of the airbnb standard. I think we can move that eslint section up here and get rid of the secondary heading.